diff --git a/node_modules/lsat-js/dist/lsat.js b/node_modules/lsat-js/dist/lsat.js
index 2d4f81c..2c145d4 100644
--- a/node_modules/lsat-js/dist/lsat.js
+++ b/node_modules/lsat-js/dist/lsat.js
@@ -216,7 +216,7 @@ class Lsat extends bufio.Struct {
      * @returns {string}
      */
     toToken() {
-        return `LSAT ${this.baseMacaroon}:${this.paymentPreimage || ''}`;
+        return `L402 ${this.baseMacaroon}:${this.paymentPreimage || ''}`;
     }
     /**
      * @description Converts LSAT into a challenge header to return in the WWW-Authenticate response
@@ -227,7 +227,7 @@ class Lsat extends bufio.Struct {
     toChallenge() {
         assert(this.invoice, `Can't create a challenge without a payment request/invoice`);
         const challenge = `macaroon="${this.baseMacaroon}", invoice="${this.invoice}"`;
-        return `LSAT ${challenge}`;
+        return `L402 ${challenge}`;
     }
     toJSON() {
         return {
@@ -371,4 +371,4 @@ class Lsat extends bufio.Struct {
     }
 }
 exports.Lsat = Lsat;
-Lsat.type = 'LSAT';
+Lsat.type = 'L402';
diff --git a/node_modules/lsat-js/src/lsat.ts b/node_modules/lsat-js/src/lsat.ts
index bee64b2..13226e7 100644
--- a/node_modules/lsat-js/src/lsat.ts
+++ b/node_modules/lsat-js/src/lsat.ts
@@ -50,7 +50,7 @@ export class Lsat extends bufio.Struct {
   routingFeePaid: number | null
   invoiceAmount: number
 
-  static type = 'LSAT'
+  static type = 'L402'
 
   constructor(options: LsatOptions) {
     super(options)
@@ -242,7 +242,7 @@ export class Lsat extends bufio.Struct {
    * @returns {string}
    */
   toToken(): string {
-    return `LSAT ${this.baseMacaroon}:${this.paymentPreimage || ''}`
+    return `L402 ${this.baseMacaroon}:${this.paymentPreimage || ''}`
   }
 
   /**
@@ -257,7 +257,7 @@ export class Lsat extends bufio.Struct {
       `Can't create a challenge without a payment request/invoice`
     )
     const challenge = `macaroon="${this.baseMacaroon}", invoice="${this.invoice}"`
-    return `LSAT ${challenge}`
+    return `L402 ${challenge}`
   }
 
   toJSON(): LsatJson {
